laa_court_data_api_app/internal/oauth_client.py

```diff
@@ -0,0 +1,70 @@
+import datetime as dt
+import logging
+
+import httpx
+from fastapi import Depends
+
+from ..config import court_data_adaptor as cda
+from ..models.token_response import TokenResponse
+
+logger = logging.getLogger(__name__)
```

Comment:
I should have mentioned this , i think the package needs to be listed in our logger config file in our root. this is so that the log can describe where the log came from in our app

Reply:
The testing i did showed the logs coming through as we have the base package logging `laa_court_data_adaptor` so any submodules should get picked up from this. Will monitor as we move forward though to ensure this is the case.